setup/main.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
```

Comment:
Looks like a template rather than an actual licence. The repo already includes the licence and I don't think we have to add it here.

Reply:
right, that was generated by the Cobra CLI and I will remove it

Reply:
removed in b307d9e